libgalois/include/katana/DynamicBitset.h

```diff
@@ -35,15 +35,15 @@
 
 #include "katana/AtomicWrapper.h"
 #include "katana/Galois.h"
-#include "katana/PODResizeableArray.h"
+#include "katana/PodVector.h"
 #include "katana/config.h"
 
 namespace katana {
 /**
  * Concurrent dynamically allocated bitset
  **/
 class KATANA_EXPORT DynamicBitset {
-  katana::PODResizeableArray<katana::CopyableAtomic<uint64_t>> bitvec_{};
+  katana::PodVector<katana::CopyableAtomic<uint64_t>> bitvec_{};
```

Comment:
slight preference for `PODVector`. We have `AtomicPODProperty` for instance already.

And for consistency, probably `NUMAArray`

Reply:
Yes, will do.